hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java

```diff
@@ -215,4 +217,16 @@ public String fetchSingleMessageFromPrimary(String brokersClusterName, TopicName
                 .map(p -> new MessageTextPreview(new String(p.getContent(), StandardCharsets.UTF_8), p.isTruncated()))
                 .collect(toList());
     }
+
+    public List<TopicNameWithMetrics> getTopicsMetrics(Query<TopicNameWithMetrics> query) {
+        return query.filter(
+                getAllTopics()
+                        .stream()
+                        .map(t -> {
```

Comment:
A private method doing this map() logic here would be nice so we could method-reference it